test/dotnet-uninstall.Tests/Shared/Filterers/MajorMinorOptionFiltererTests.cs

```diff
@@ -208,7 +127,8 @@ internal void TestMajorMinorOptionFiltererGood(IEnumerable<Bundle> testBundles,
         [InlineData("3.0.100-preview5-011568")]
         internal void TestMajorMinorOptionFiltererInvalidInputVersionStringException(string argValue)
         {
-            TestFiltererException<InvalidInputVersionException>(DefaultTestBundles, argValue, BundleType.Sdk | BundleType.Runtime, DefaultTestArchSelection);
```

Comment:
Is there a test for "when options have both --sdk and --runtime, it should throw"? If so just point me to it and it is good  to merge. If not, just add it and merge

Reply:
Yes, it's in the base test class and applies to all filterer tests.